### DISCUSSION TITLE
PiJuice + RPi zero 2 w for BirdNet Pi

### DISCUSSION THREAD

**Discussion by Keizeus** [PiSupply/PiJuice#984] [Q&A]
I have BirdNet Pi installed and running on my RPi zero 2 w.

It works when powered by the wall plug. When I power it with my PiJuice zero via the 12,000mAH battery, it does nothing. The pi juice is connected to the RPi zero 2 w via gpio pins. I am running the lite version of the os on the RPi. I have the pi juice software installed on the RPi.

The green light turns on for the RPi, it gets hot but that's it. Doesn't appear to connect to wifi or use USB.

What do I need to do to fix this?
PS, please respond in explain-like-I'm-5 format because I'm quite the noob when it comes to RPi.

**Comment by tvoverbeek** [comment]
Are you sure the 40 pin connector is correctly connected? Not shifted by one row?

**Reply by Keizeus** [reply]
I just checked it. It's lined up. The only thing wonky is I made the pi juice zero an... under-hat? A sock?

All the pins hit the same but the RPi is on top because it gets hot and needs a heat sink.
